justfile

```diff
@@ -1,9 +1,17 @@
 # Use bash with strict flags
 set shell := ["bash", "-euo", "pipefail", "-c"]
 
+# CI/output mode detection (inspired by monorepo pattern)
+ci := env("CI", "false")
```

Comment:
this is an actual environment variable, right?  Maybe use a prefix on the env var name to reduce risk of collisions in the global namespace?